rapid_pro_tools/rapid_pro_client.py

```diff
@@ -277,6 +277,55 @@ def send_message_to_urn(self, message, target_urn, interrupt=False):
         log.info(f"Message send request created with broadcast id {response.id}")
         return response.id
 
+    def send_message_to_urns(self, message, target_urns, interrupt=False):
+        """
+        Sends a message to URNs.
+
+        :param message: Text of the message to send.
+        :type message: str
+        :param target_urn: URNs to send the message to.
+        :type target_urn: str
+        :param interrupt: Whether to interrupt the target_urn from flows before sending the message.
+        :type interrupt: bool
+        :return: Ids of the Rapid Pro broadcasts created for this send request.
+                 These ids may be used to check on the status of the broadcast by making further requests to Rapid Pro.
+        :rtype: list of int
+        """
+        log.info(f"Sending a message to {len(urns)} URNs...")
+        log.debug(f"Sending to {urns}...")
+        batch = []
+        ids = []
+        interrupted = 0
+        sent = 0
+
+        for urn in urns:
+            batch.append(urn)
+            if len(batch) >= 100:  # limit of 100 imposed by Rapid Pro's API
+                if interrupt:
+                    self.rapid_pro.bulk_interrupt_contacts(batch)
+                    interrupted += len(batch)
+                    log.info(f"Interrupted {interrupted} / {len(urns)} URNs")
+
+                response: Broadcast = self.rapid_pro.create_broadcast(message, urns=batch)
```

Comment:
I wonder if we should remove the : Broadcast to avoid future confusion, as we're not using type hinting anywhere else

Reply:
Will do